src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -77,30 +49,36 @@ public void excercise1() {
         webDriver.switchTo().parentFrame();
 
         // 11. Assert that there are 5 items in the Left Section are displayed and they have proper text
-        softAssert.assertEquals(webDriver.findElement(By.xpath("//span[contains(text(), 'Home')]"))
-                                         .getText(), "Home");
-        softAssert.assertTrue(webDriver.findElement(By.xpath("//span[contains(text(), 'Home')]")).isDisplayed());
-
-        softAssert.assertEquals(webDriver.findElement(By.xpath("//span[contains(text(), 'Contact form')]"))
-                                         .getText(), "Contact form");
-        softAssert.assertTrue(webDriver.findElement(By.xpath("//span[contains(text(), 'Contact form')]"))
-                                       .isDisplayed());
+        List<WebElement> actualSideNavBarItems =
+            webDriver.findElements(By.xpath("//ul[@class='sidebar-menu left']/li/a/span"));
+        assertSideNavBarItemText(actualSideNavBarItems);
 
-        softAssert.assertEquals(webDriver.findElement(By.xpath("//span[contains(text(), 'Service')]"))
-                                         .getText(), "Service");
-        softAssert.assertTrue(webDriver.findElement(By.xpath("//span[contains(text(), 'Service')]")).isDisplayed());
+        softAssert.assertAll();
+    }
 
-        softAssert.assertEquals(webDriver.findElement(By.xpath("//span[contains(text(), 'Metals & Colors')]"))
-                                         .getText(), "Metals & Colors");
-        softAssert.assertTrue(webDriver.findElement(By.xpath("//span[contains(text(), 'Metals & Colors')]"))
-                                       .isDisplayed());
+    private void assertHeaderNavBarItemText(List<WebElement> actualHeaderNavBarItems) {
+        for (int i = 0; i < actualHeaderNavBarItems.size(); i++) {
+            softAssert.assertEquals(actualHeaderNavBarItems.get(i).getText(),
+                DataClass.expectedHeaderNavBarItemTexts.get(i));
```

Comment:
Commonly for such verification type try to check expected actualHeaderNavBarItems size and then verify items using assert on actualHeaderNavBarItems.containsAll(expectedHeaderNavBarItemTexts)